pom.xml

```diff
@@ -4,13 +4,40 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>io.github.java-native.jssc</groupId>
+  <groupId>io.github.java-native</groupId>
   <artifactId>jssc</artifactId>
-  <version>2.9.1</version>
+  <version>2.9.2-SNAPSHOT</version>
```

Comment:
As the version 2.9.1 is considered released, but not up to Maven Central standards, I'd suggest bumping it to 2.9.2, and keeping the SNAPSHOT suffix between releasable revisions..